src/main/java/com/breskul/hw/controllers/NasaController.java

```diff
@@ -0,0 +1,78 @@
+package com.breskul.hw.controllers;
+
+import com.breskul.hw.controllers.entity.Photos;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.springframework.http.HttpStatus.MOVED_PERMANENTLY;
+
+@Component
+public class NasaController {
+
+    Logger logger = LoggerFactory.getLogger(NasaController.class);
+
+    final String url = "https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos";
```

Comment:
It is a good practice to store such values in a properties file, this allows you to reuse them in different places, store all properties in responsive files and keep code clear and readable.

Reply:
but if you want to keep it here as a constant, please make it `private static` and use the name according to the java naming convention(URL)